src/elements/TabbedContainer/Tab.js

```diff
@@ -0,0 +1,31 @@
+import './TabbedContainer.less';
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Div from '../Div/Div';
+
+export default class Tab extends Component {
+	render() {
+		const classes = classNames(
+			'tab-panel',
+			this.props.active ? ' active' : ''
```

Comment:
If using `classnames` package, you don't need to include a leading space for the class name. Also, the `classnames` API allows you to more simply trigger boolean classnames by passing an object where the keys are classes, and the values are booleans (`classnames` also accepts unlimited arguments and types):

```
const classes = classNames('tab-panel', {active: props.active});
```

Reply:
Nice, I didn't even know that. I always just do the short-circuit style: `this.props.active && 'active'`

Reply:
Yeah, `classnames` is one of the more powerful yet simple packages out there for react. You can pass arrays, too https://github.com/JedWatson/classnames#usage